superbench/benchmarks/micro_benchmarks/cuda_decode_performance/CMakeLists.txt

```diff
@@ -4,114 +4,121 @@
 cmake_minimum_required(VERSION 3.18)
 project(cuda_decode_performance)
 
-find_package(CUDA QUIET)
-if(CUDA_FOUND)
-  set(CMAKE_CXX_STANDARD 17)
-  set(CMAKE_CXX_STANDARD_REQUIRED ON)
-
-  set(THIRD_PARTY_SAMPLE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../../../third_party/Video_Codec_SDK/Samples)
-  set(NVCODEC_PUBLIC_INTERFACE_DIR ${THIRD_PARTY_SAMPLE_DIR}/../Interface)
-  set(NVCODEC_UTILS_DIR ${THIRD_PARTY_SAMPLE_DIR}/Utils)
-  set(NV_CODEC_DIR ${THIRD_PARTY_SAMPLE_DIR}/NvCodec)
-  set(NV_DEC_DIR ${THIRD_PARTY_SAMPLE_DIR}/NvCodec/NvDecoder)
-
-  if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
-    find_package(PkgConfig REQUIRED)
-    pkg_check_modules(PC_AVCODEC REQUIRED IMPORTED_TARGET libavcodec)
-    pkg_check_modules(PC_AVFORMAT REQUIRED IMPORTED_TARGET libavformat)
-    pkg_check_modules(PC_AVUTIL REQUIRED IMPORTED_TARGET libavutil)
-    pkg_check_modules(PC_SWRESAMPLE REQUIRED IMPORTED_TARGET libswresample)
-
-    set(NV_FFMPEG_HDRS ${PC_AVCODEC_INCLUDE_DIRS})
-    find_library(AVCODEC_LIBRARY NAMES avcodec
-    HINTS
-    ${PC_AVCODEC_LIBDIR}
-    ${PC_AVCODEC_LIBRARY_DIRS}
+
+ # Check architecture
+ if(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64|arm64")
+    message(WARNING "Skipping Cuda decode Performance build. This build only supports x86_64 arch.")
+ else()
```

Comment:
could you fix the indents here? seems there's one extra space at the beginning of each line

Reply:
Done